File: app/src/main/res/ui/newPassword/NewPasswordActivity.java
package com.khedr_and_basuoin.deliveryapp.ui.newPassword;

import androidx.appcompat.app.AppCompatActivity;
import androidx.databinding.DataBindingUtil;

import android.app.Activity;
import android.os.Bundle;

import com.khedr_and_basuoin.deliveryapp.R;
import com.khedr_and_basuoin.deliveryapp.databinding.ActivityNewPasswordBinding;

public class NewPasswordActivity extends AppCompatActivity {
    ActivityNewPasswordBinding b;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        b= DataBindingUtil.setContentView(this,R.layout.activity_new_password);

    }
}

File: app/src/main/java/com/khedrandbasuoni/baytifood/presentaion/ui/newPassword/NewPasswordActivity.kt
package com.khedrandbasuoni.baytifood.presentaion.ui.newPassword

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.databinding.DataBindingUtil
import com.khedrandbasuoni.baytifood.R
import com.khedrandbasuoni.baytifood.databinding.ActivityNewPasswordBinding

class NewPasswordActivity : AppCompatActivity() {
    var b: ActivityNewPasswordBinding? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        b = DataBindingUtil.setContentView(this, R.layout.activity_new_password)
    }
}